### ISSUE TITLE
[1.18.2] Cant open Hopper in Multiplayer 

### ISSUE THREAD

**Issue by pewblaze** [pcal43/copper-hopper#24]
Hi i cant open the Copper Hopper in Multiplayer. Ill get an error:

[07:46:26] [Server thread/ERROR]: Failed to handle packet net.minecraft.class_2885@24ca1bd7, suppressing error
java.lang.NullPointerException: null
	at Not Enough Crashes deobfuscated stack trace.(1.18.2+build.3) ~[?:?]
	at java.util.Objects.requireNonNull(Unknown Source) ~[?:?]
	at net.pcal.copperhopper.CohoService.getScreenHandlerType(CohoService.java:72) ~[copperhopper-0.0.7+1.18.2.jar:?]
	at net.pcal.copperhopper.CohoScreenHandler.getType(CohoScreenHandler.java:20) ~[copperhopper-0.0.7+1.18.2.jar:?]
	at net.minecraft.server.network.ServerPlayerEntity.openHandledScreen(ServerPlayerEntity:1038) ~[?:?]
	at net.minecraft.block.HopperBlock.onUse(HopperBlock:140) ~[?:?]
	at net.minecraft.block.AbstractBlock$AbstractBlockState.onUse(AbstractBlock:939) ~[?:?]
	at net.minecraft.server.network.ServerPlayerInteractionManager.redirect$bff000$activateWithOptionalCactus(ServerPlayerInteractionManager:2029) ~[?:?]
	at net.minecraft.server.network.ServerPlayerInteractionManager.interactBlock(ServerPlayerInteractionManager:336) ~[?:?]
	at net.minecraft.server.network.ServerPlayNetworkHandler.onPlayerInteractBlock(ServerPlayNetworkHandler:1081) ~[?:?]
	at net.minecraft.network.packet.c2s.play.PlayerInteractBlockC2SPacket.apply(PlayerInteractBlockC2SPacket:30) ~[?:?]
	at net.minecraft.network.packet.c2s.play.PlayerInteractBlockC2SPacket.apply(PlayerInteractBlockC2SPacket:8) ~[?:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils:22) ~[?:?]
	at net.minecraft.server.ServerTask.run(ServerTask:18) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor:157) ~[?:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor:23) ~[?:?]
	at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer:808) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer:165) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor:131) ~[?:?]
	at net.minecraft.server.MinecraftServer.runOneTask(MinecraftServer:790) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.runTask(MinecraftServer:784) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor:116) ~[?:?]
	at net.minecraft.server.MinecraftServer.runTasksTillTickEnd(MinecraftServer:768) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.handler$bch000$modifiedRunLoop(MinecraftServer:4405) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:676) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:273) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Unknown Source) [?:?]

Full log:
[latest.log](https://github.com/pcal43/copper-hopper/files/8747356/latest.log)

We did change the lithium config.
Not sure what happens ^^"

**Comment by dragonmaus** [comment]
I'm getting a similar error:
```
[23:29:22] [Server thread/ERROR]: Failed to handle packet net.minecraft.class_2885@2a3bb86a, suppressing error
java.lang.NullPointerException: null
        at java.util.Objects.requireNonNull(Unknown Source) ~[?:?]
        at net.pcal.copperhopper.CohoService.getScreenHandlerType(CohoService.java:72) ~[copperhopper-0.0.7+1.18.2.jar:?]
        at net.pcal.copperhopper.CohoScreenHandler.method_17358(CohoScreenHandler.java:20) ~[copperhopper-0.0.7+1.18.2.jar:?]
        at net.minecraft.class_3222.method_17355(class_3222.java:1038) ~[server-intermediary.jar:?]
        at net.minecraft.class_2377.method_9534(class_2377.java:140) ~[server-intermediary.jar:?]
        at net.minecraft.class_4970$class_4971.method_26174(class_4970.java:939) ~[server-intermediary.jar:?]
        at net.minecraft.class_3225.method_14262(class_3225.java:336) ~[server-intermediary.jar:?]
        at net.minecraft.class_3244.method_12046(class_3244.java:1081) ~[server-intermediary.jar:?]
        at net.minecraft.class_2885.method_12547(class_2885.java:30) ~[server-intermediary.jar:?]
        at net.minecraft.class_2885.method_11054(class_2885.java:8) ~[server-intermediary.jar:?]
        at net.minecraft.class_2600.method_11072(class_2600.java:22) ~[server-intermediary.jar:?]
        at net.minecraft.class_3738.run(class_3738.java:18) ~[server-intermediary.jar:?]
        at net.minecraft.class_1255.method_18859(class_1255.java:157) ~[server-intermediary.jar:?]
        at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:808) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:165) ~[server-intermediary.jar:?]
        at net.minecraft.class_1255.method_16075(class_1255.java:131) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:790) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:784) ~[server-intermediary.jar:?]
        at net.minecraft.class_1255.method_18857(class_1255.java:140) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:769) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:701) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:273) ~[server-intermediary.jar:?]
        at java.lang.Thread.run(Unknown Source) [?:?]
```

Lithium config is set properly, Polymer support not enabled and no trace of Polymer anywhere, mod is installed on both client and server.

**Comment by pcal43** [comment]
should be fixed in 0.0.8.
